tests/routes/articles.test.js

```diff
@@ -167,7 +167,7 @@ describe('Articles API', () => {
       const response = await chai.request(app).get(`${baseRoute}/articles?author=Lagosplk`);
 
       response.should.have.status(404);
-      response.body.message.should.equal('Lagosplk Not found');
+      response.body.error.message.should.equal('No articles found');
```

Comment:
This response should be an object that has data as an empty array, not an object that has an error message. Am I missing something?

Reply:
From the response body, it was returning an error message of 'No articles found' but I just had synced with Nenny so she told me there is no need of the test as she has updated her controller. So I will be removing this test.